lib/new-admin/transactions.js

```diff
@@ -76,6 +78,46 @@ function batch (from = new Date(0).toISOString(), until = new Date().toISOString
   ]).then(packager)
 }
 
+function simplifiedBatch (from = new Date(0).toISOString(), until = new Date().toISOString(), limit = null, offset = 0) {
+  return batch(from, until, limit, offset)
+    .then(res => {
+      const fields = ['txClass', 'id', 'created', 'machineName',
+        'cryptoCode', 'fiat', 'fiatCode', 'phone', 'toAddress',
+        'txHash', 'dispense', 'error', 'status', 'fiatProfit', 'cryptoAmount']
+      return _.map(_.compose(_.pick(fields), getStatus, getProfit, getCryptoAmount))(res)
+    })
+}
+
+const getCryptoAmount = it => { return { ...it, cryptoAmount: coinUtils.toUnit(BN(it.cryptoAtoms), it.cryptoCode).toString() } }
+
+const getProfit = it => {
+  const getCommissionFee = it => Number.parseFloat(it.commissionPercentage) * Number.parseFloat(it.fiat)
```

Comment:
It's probably better to use BigNumber here

Reply:
hmm, we're doing similar calculations at the front-end level (for the system performance panel I believe) and we use Number. we better set a standard for this, right?

Reply:
Yeah, everything financial related should be BigNumber